packages/docusaurus-plugin-ideal-image/package.json

```diff
@@ -10,8 +10,8 @@
   "dependencies": {
     "@endiliey/lqip-loader": "^3.0.0",
     "@endiliey/responsive-loader": "^1.3.1",
-    "react-ideal-image": "^0.0.5",
-    "react-waypoint": "8.1.0",
+    "@endiliey/react-ideal-image": "^0.0.8",
```

Comment:
Forked because that repo is so outdated.... :( (it still uses babel 7). Maintainer don't want to respond to issues and PR as well. My update is minimal https://github.com/stereobooster/react-ideal-image/compare/master...endiliey:master

Interestingly with this fork main bundle size reduced by 14kbs. Crazy ...
<img width="956" alt="after 221kb" src="https://user-images.githubusercontent.com/17883920/68780864-773c6180-0669-11ea-8af2-751bd6a23964.PNG">


<img width="959" alt="207kb after  wew" src="https://user-images.githubusercontent.com/17883920/68780883-80c5c980-0669-11ea-97cc-334c9b57beac.PNG">